### PR TITLE
Added option to approve multiple contest in Admin Dashboard

### LINKED ISSUE #161: Approve contests in Admin
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in our slack channel at https://opensourcehelp.slack.com/. Request invite [here](https://opensourcehelp.herokuapp.com)

**Expected Behavior**:
Admin have to open each contests in order to approve contest in Admin dashboard 

**Actual Behavior**:
Admin should have option to slect multiple contests and approve at once

**Would you like to work on this issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [ ] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: fixes #161 

Brief description of what this PR does.
Admin now can select multiple contests and approve at once

Screenshot:
image

### CODE REVIEW

File: oshc/main/admin.py [hunk 1]
```diff
@@ -10,5 +10,18 @@ class chatSessionAdmin(admin.ModelAdmin):
...
+        if contest_approved == 1:
+            message_bit = "1 contest"
+        else:
+            message_bit = "{} contests were".format(contest_approved)
+        self.message_user(request, "{} approved.".format(message_bit))
+    approve_contest.short_description = "Approve"
+
 admin.site.register(chatSession, chatSessionAdmin)
-admin.site.register(Contest)
+admin.site.register(Contest, contestAdmin)
```
Comment (tapaswenipathak) [inline oshc/main/admin.py:27]: Aah you are doing this in parts? :) I will close that one then. We can have a reject as well.

**Review by tapaswenipathak — APPROVED** [review]
works! merging this. Thanks! 🚀

File: oshc/main/admin.py (hunk 1 above)
Comment (vaibhavsingh97) [reply oshc/main/admin.py:27]: @tapasweni-pathak I think I failed to express why I opened the PR you closed... actually what I think that before Approval we will look at the authenticity of contest so to do that you have to click on each contest if user had submitted 10 contests .....so I made this process easy by displaying all the fields in dashboard only. Let me know your thoughts

Comment (tapaswenipathak) [reply oshc/main/admin.py:27]: This one also has those changes (#160), that's why I closed that one.

In #160 you enabled the display of all the fields on the initial page. In this one you enabled the display as well as approval. Am I missing something here? 😕

Comment (vaibhavsingh97) [reply oshc/main/admin.py:27]: @tapasweni-pathak I am really sorry!! 😅😅 I don't how this happened...and yes thanks for merging.
While sending PR I thought, I had sent two PR for different issues...and this was mine misunderstanding 😅😅

**Pull request merged**